mnist_simple/Makefile

```
@@ -0,0 +1,33 @@
+# This is a simple Makefile used to build the example source code.
+# This example might requires some modifications in order to work correctly on
+# your system.
+# If you're not using the Armadillo wrapper, replace `armadillo` with linker commands
+# for the BLAS and LAPACK libraries that you are using.
+
+TARGET := mnist_simple
+SRC := mnist_simple.cpp
+LIBS_NAME := armadillo mlpack boost_serialization
+
+CXX := g++
+CXXFLAGS := -std=c++11 -Wall -Wextra
+LDFLAGS  := -fopenmp
+DBGFLAG := -g
+INCLFLAGS := -I . # Add header directories if there is any.
+
+OBJS := $(SRC:.c=.o)
```

```suggestion
OBJS := $(SRC:.cpp=.o)
```
Not sure, maybe this is needed?  If this is added, then it looks like some other changes are needed so that the `CXXFLAGS` are right for the original compilation.  Anyway, the existing way seems to work; it just skips the compilation to `.o` (which is fine) and compiles directly to the executable.